tests/server.test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+const supertest = require('supertest');
+const { app } = require('../server/index.js');
+
```

Comment:
You could mock your db and assert that the correct queries are run.

You should not need to run the server here to run tests.